### ISSUE TITLE
GTF to Refflat tool for use with CollectRnaSeqMetrics

### ISSUE THREAD

**Issue by nh13** [broadinstitute/picard#805]
Since most folks use GTFs for RNASeq, but `CollectRnaSeqMetrics` uses a refflat file, it would be nice to have a tool that can convert GTF to refflat, and then we can update `CollectRnaSeqMetrics` to take in either a refflat or GTF file.

This an example method (requires Jim Kent's tools):
```
 gtfToGenePred \
    -genePredExt \
    -geneNameAsName2 \
    -ignoreGroupsWithoutExons \
    !{params.gtf} \
    /dev/stdout | \
    awk 'BEGIN { OFS="\t"} {print $12, $1, $2, $3, $4, $5, $6, $7, $8, $9, $10}' 
```

Also, it wouldn't be hard to pull out the records with `gene_type` equal to `rRNA` to create the requisite ribosomal intervals.

**Comment by sooheelee** [comment]
Commented prematurely. When you say tool, you mean something under the hood that CollectRnaSeqMetrics can call upon, not a outward facing tool.

**Comment by nh13** [comment]
@sooheelee yes, making `CollectRnaSeqMetrics` accept a refflat file would be great; I think I mentioned that above: "then we can update CollectRnaSeqMetrics to take in either a refflat or GTF file."  Nonetheless, if we are going to support a GTF as input to `CollectRnaSeqMetrics`, then 95% of the work would be done, and we could _also_ have a really useful tool.

**Comment by nh13** [comment]
@yfarjoun thoughts?

**Comment by yfarjoun** [comment]
are you volunteering? I have no objections to having the tool accept GTF as input.

**Comment by nh13** [comment]
@yfarjoun maybe, but not at the moment, I think it is very mechanical and would be a great exercise for a student project if you have any students?

**Comment by yfarjoun** [comment]
interesting idea. I'll run it by folks that might want to cut their teeth on some Picard stuff.

**Comment by royfrancis** [comment]
Just checking up on this 7 years later. So, can `CollectRnaSeqMetrics` take a GTF file?

**Comment by serge2016** [comment]
Same question here. Any progress?
